src/handler/MethodNameInflector/HandleClassNameInflector.js

```diff
@@ -0,0 +1,11 @@
+import MethodNameInflector from './MethodNameInflector';
+
+export default class HandleClassNameInflector extends MethodNameInflector {
+    constructor(methodName) {
+        super();
+        this.methodName = methodName || "handle";
```

Comment:
According to this class, https://github.com/thephpleague/tactician/blob/master/src/Handler/Mapping/MethodName/HandleLastPartOfClassName.php, seems like it does not need the property `this.methodName`, so please remove and use `handle` hardcoded in the `inflect` method.

Reply:
Removed this.methodName and used handle hardcoded

Reply:
Cool, Thanks!